SimPEG/Examples/MT_1D_analytic_nlayer_Earth.py

```diff
@@ -0,0 +1,428 @@
+from scipy.constants import epsilon_0, mu_0
+import matplotlib.pyplot as plt
+import numpy as np
+from ipywidgets import *
+from SimPEG.EM.Utils import k, omega
+
+"""
+MT1D: n layered earth problem
+*****************************
+
+Author: Thibaut Astic
+Contact: thast@eos.ubc.ca
+Date: January 2016
```

Comment:
Also, @thast : I am not sure if you made updates to this code that you would like to put in this pull request before it gets onto dev. Up to you!